### DISCUSSION TITLE
Pico HSM and Windows Server Certification Authority

### DISCUSSION THREAD

**Discussion by ko-gh** [polhenarejos/pico-hsm#70] [Q&A]
Hello to all,

would it be possible to use the Pico HSM as a Cypher / Key Storage Provider for a Windows Server Certification Authority to generate and store the Root CA Keys?

e.g. if I use a RP2350 via WiFi?

Kind regards

**Comment by polhenarejos** [comment]
Pico HSM can be interfaced with PKCS#11. If your system accepts it, then yes.
